testing/botan/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Contributor:
+# Maintainer:
+pkgname=botan
+_pkgreal=Botan
+pkgver=2.4.0
+pkgrel=0
+pkgdesc="Crypto and TLS for C++11"
+url="https://botan.randombit.net/"
+arch="x86_64 x86"
+license="BSD-2-Clause"
+depends=""
+depends_dev="python2 boost-dev libressl-dev sqlite-dev bzip2-dev zlib-dev"
+makedepends="$depends_dev"
+subpackages="$pkgname-dev $pkgname-doc"
+source="https://botan.randombit.net/releases/$_pkgreal-$pkgver.tgz"
+builddir="$srcdir/$_pkgreal-$pkgver"
+install=""
+
+build() {
+	cd "$builddir"
+	./configure.py \
+		--prefix=/usr \
+		--mandir=/usr/share/man \
+		--with-boost \
+		--with-openssl \
+		--with-sqlite3 \
+		--with-bzip2 \
+		--with-zlib
+	make
+}
+
+check() {
+	cd "$builddir"
+
+	LD_LIBRARY_PATH=. python src/python/botan2.py
```

Comment:
Please fix `python` to `python2`.

Reply:
Sure. Why prefer one symbolic link over the other?

Reply:
`python` may be changed to python3 in the future. `python2` makes is more clear that it uses Legacy Python.